assets/app/view/game/round/operating.rb

```diff
@@ -34,8 +35,10 @@ def render
           entity = entity.owner if entity.company? && !round.active_entities.one?
 
           left = []
+          left << render_mode_button if @step.respond_to?(:mode_enabled?) && @step.mode_enabled?
           left << h(SpecialBuy) if @current_actions.include?('special_buy')
-          left << h(RouteSelector) if @current_actions.include?('run_routes')
+          left << h(RouteSelector) if @current_actions.include?('run_routes') && !@step.alternate_selector?
```

Comment:
switch this to an if else?

Reply:
why would you call it alternate_selector when it's track conversion

Reply:
OK, bad name. It was an alternative to the normal selector, but it still allows one to select a route. I'll come up with a better name.

Reply:
Switching to a flat if/else is difficult since I need to check for "run_routes" before I know I can call the alternate_selector (or whatever) method.